src/js/hca/config/form.js

```diff
@@ -124,9 +123,7 @@ const formConfig = {
   migrations,
   savedFormMessages: {
     notFound: 'Please start over to apply for health care.',
-    noAuth: 'Please sign in again to resume your application for health care.',
-    success: resumeMessage,
-    startOver: 'This will remove anything you have put into the Health Care Application.'
```

Comment:
Do we want to remove this custom start over text? That shows up in the start over modal, I think?

Reply:
Thanks for pointing that out @jbalboni, @stacyhlinn @peggygannon could you please share your thoughts on whether the HCA start over model should have a custom modal message, or whether it should use the generic message that the rest of the forms use?
Generic
<img width="801" alt="screen shot 2017-09-14 at 11 13 51 am" src="https://user-images.githubusercontent.com/16051172/30446617-85eeab92-993e-11e7-9c16-5ca5eedde8d0.png">

Current
<img width="748" alt="screen shot 2017-09-14 at 11 17 21 am" src="https://user-images.githubusercontent.com/16051172/30446601-7ad474c6-993e-11e7-9cb0-89382dcf90eb.png">

Reply:
The more specific one (current version) seems less confusing to me. Would rely on @peggygannon to make the right judgement though.

Reply:
It would be nice to not have custom text here, from a technical perspective, since we have to remember to pull it in in several places now.

Reply:
Makes sense! A generic one would work too. Just want to make sure Peggy has seen and reviewed the message.

Reply:
@jbalboni @stacyhlinn @cehsu I think we should use the generic message, as we do for other forms. 
--Can we please make the below edit to the bold sentence? ("delete" is more clear/direct than "erase"--which reminds me of school chalkboard!) 
--Can this edited message be applied across all the forms? 

Starting over will delete your in-progress form. 
[[rest of message is OK as is]]

Reply:
Thanks for the insights @peggygannon, we can make that change, and it will be on a shared component that all of the forms use.